core/arch/proc.h

```diff
@@ -296,12 +296,13 @@ extern size_t cache_line_size;
 
 /* xcr0 and xstate_bv feature bits */
 enum {
-    XCR0_HI16_ZMM = 7,
-    XCR0_ZMM_HI256 = 6,
-    XCR0_OPMASK = 5,
-    XCR0_AVX = 4,
-    XCR0_SSE = 2,
-    XCR0_FP = 1,
+    XCR0_HI16_ZMM = 0x80,
```

Comment:
Wait, so this was just a list, but now it's the actual bits used by the processor?  Maybe the comment above should clarify that.

Reply:
I've augmented the comment. Yes, it's always been the actual bit.

Reply:
> it's always been the actual bit.

But then it was buggy before?  XCR0_HI16_ZMM used to be 7.

Reply:
Yes. But the XCR0 bits are always set in a way that it didn't show up.